SU-HSDO/suhumsci
# Hotfix(STN-843): adjust top positioning of mobile nav button

## Changes
The Top positioning applied to the mobile nav button (`humsci_basic/src/scss/components/_main-nav.scss`) only accounted for situations when a search block was present. Utilizing the `sibling selector` in CSS we can adjust the `Top` when `nav button` is a sibling of `.su-site-search `.

## Issue
[STN-843](https://sparkbox.atlassian.net/browse/STN-843)

# READY FOR REVIEW

## Summary
Default the Top positioning to -4.675rem and increase it to -12rem when
nav button is a sybling of .su-site-search

## Need Review By (Date)
asap

## Urgency
'medium'

## Steps to Test
1. Run `npm run test` within the `humsci_basic` directory.
2. Add a Search Block and after viewing the site, confirm that Mobile Menu Button never makes it way up into the brand bar.
3. Disable Search Block and after viewing the site, confirm that Mobile Menu Button never makes it way up into the brand bar.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
